solutions.js

```diff
@@ -1,5 +1,51 @@
-console.log('Defining functions!');
+function changeBackground(){
+    var navBar = document.querySelector('#topbar');
+    var secBar = document.querySelector('#secondary-nav');
+    navBar.style.backgroundColor = 'red';
+    secBar.style.backgroundColor = 'red';
+
+
+    console.log('i rule the navBAR')
+};
+changeBackground();
+
+function changeH1(){
+var shout = document.querySelector('#user-gallery-message');
+        shout.innerText='This website is now under Miguel\'s control';
+};
+
+changeH1();
+
+
+
+var imageLink = [...document.querySelectorAll('.image-list-link')];
+
+for (var i=0;i<imageLink.length;i++){
+        imageLink[i].addEventListener("click",function(e){
+        alert("PLEASE?!?!?!?");
+        //prevent event action
+        e.preventDefault();
```

Comment:
You'll want to prevent the default action FIRST, then prompt the user to type please.